cli/src/commands/bookmark/list.rs

```diff
@@ -240,9 +242,17 @@ pub fn cmd_bookmark_list(
         }
     }
 
+    let sort_keys = if args.sort.is_empty() {
+        let SortKeys(sort_keys) = workspace_command
+            .settings()
+            .get_value_with("ui.bookmark-list-order", TryInto::try_into)?;
```

Comment:
nit: better to mention the config key in `--sort` help.

It would be nice if we can use "sort" instead of "order" consistently, but I don't have a good suggestion.

Reply:
> nit: better to mention the config key in --sort help.

Good point!

> It would be nice if we can use "sort" instead of "order" consistently, but I don't have a good suggestion.

Yes! Changed it to `bookmark-list-sort-keys`